vunit.dockerfile

```diff
@@ -1,28 +1,49 @@
 ARG TAG="buster-mcode"
 
+#---
+
 FROM ghdl/ghdl:$TAG AS base
+
 ARG PY_PACKAGES
-RUN apt-get update -qq \
+
+RUN set -ux \
+ && apt-get update -qq \
  && DEBIAN_FRONTEND=noninteractive apt-get -y install --no-install-recommends \
-    ca-certificates \
-    curl \
-    make \
     python3 \
     python3-pip \
  && apt-get autoclean && apt-get clean && apt-get -y autoremove \
- && update-ca-certificates \
- && rm -rf /var/lib/apt/lists/* \
- && pip3 install -U pip setuptools wheel $PY_PACKAGES
+ && rm -rf /var/lib/apt/lists/*
+
+RUN set -ux \
```

Comment:
Please, avoid creating an additional layer. I.e., keep these commands merged with the previous `RUN` step.

Reply:
In my opinion it is beneficial to have separate steps here, since the calls are unrelated. If I want to change something about the `pip` call, I would still keep the build cache for the previous (apt install) layer. I think it is best practice to split unrelated commands to separate layers. It results in faster re-build times, and can potentially save space on disk.

Anyway, that is my personal view on the matter. If you feel differently, I will revert so that it is only one layer.

Reply:
While your point is valid as a rule of thumb, I think it does not apply in this specific case, because:

- Both steps are about "installing the minimal dependencies for being able to later add Python packages from sources".
- This image is built in CI, without any caching enabled. Therefore, there is no practical advantage.
- Should someone be to do several modifications/tests, copying and pasting a line is enough for spliting the steps the first time. Then, they can take advantage of their local cache before submitting changes.
- I expect these image to be extended by users/companies for adding custom dependencies on top of VUnit. Hence, the less noise we introduce, the best. In fact, I though about squashing them; but I decided not to do it because the base layers (Debian) would also be squashed.